### PR TITLE
IG-6439: add ability to define a list of custom check scripts

### PR DESCRIPTION
**Pull request by peterp415**
... to be installed on the nagios server.

Some spurious eol whitespace included in the diffs, sorry 'bout that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ You can add more commands by adding to, or overriding, the variable.
 The nagios services are currently in the "checks" per hostgroup variables, see defaults/main.yml.
 You can add more services by adding to, or overriding, the variable.
 
+### Nagios Custom Checks:
```
Comment (davidmnoriega) [inline README.md:60]: The readme doesn't say how to use `nagios_custom_checks` I have to read the code to see it's used as a list to tell the template module to template out some files(that come from where?).

File: tasks/custom_checks.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+# Install custom check scripts, if any.
+- name: Template out custom service checks
```
Comment (davidmnoriega) [inline tasks/custom_checks.yml:3]: This only puts a custom check, doesn't remove any should the user alter `nagios_custom_checks` later on.

File: tasks/server.yml [hunk 3]
```diff
@@ -85,10 +85,12 @@
 - name: include nrpe_client
   include: nrpe_client.yml
 
+- name: include custom check scripts
+  include: custom_checks.yml
```
Comment (davidmnoriega) [inline tasks/server.yml:89]: **out of scope thought** when do we consider pushing this role to ansible 2.4+?

**Review by davidmnoriega — CHANGES_REQUESTED** [review]

File: tasks/custom_checks.yml (hunk 2 above)
Comment (peterp415) [reply tasks/custom_checks.yml:3]: That's correct.  The set of checks to actually run is defined elsewhere which is where you can turn them on and off.

Comment (davidmnoriega) [reply tasks/custom_checks.yml:3]: Ah so you would use `nagios_custom_checks` to install scripts, and then in some other variable, say hey now use this check? That isn't in the readme. I'm now thinking that maybe the name of the variable should change to make it more clear that its for installing scripts, not controlling checks to be run against things.

**Pull request closed without merging**